public-ui/src/app/views/pm/pm.component.html

```diff
@@ -7,15 +7,7 @@ <h1>{{title}}</h1>
           class="btn btn-link"> {{item.groupName}} </button>
       </div>
     </aside>
-    <div *ngIf="loading()" class="loading white-card-box ">
-      <div class="row flex-items-xs-middle">
-        <div class="col-xs-12">
-          <span class="spinner spinner-lg hidden-sm-down">Loading ...</span>
-          <span class="spinner spinner-md hidden-md-up hidden-xs-down"></span>
-          <span class="spinner spinner-sm hidden-sm-up"></span>
-        </div>
-      </div>
-    </div>
+    <app-db-spinner [loading]="loading()"></app-db-spinner>
```

Comment:
This seems to be variable right ?

Reply:
This is how it was set up previously. i am going to leave it for now